test/files.spec.ts

```diff
@@ -112,7 +112,7 @@ describe("SmartlingFilesApi class tests.", () => {
                     .setRetrievalType(RetrievalType.PUBLISHED)
                     .enableDebugMode();
 
-                await filesApi.downloadFile(projectId, fileUri, localeId, params);
+                await filesApi.downloadFile(projectId, fileUri, localeId, true, params);
```

Comment:
Maybe pass `TEXT` here. It's not important for this test, but at least no question about what does `true` mean for `responseType`? Is it important for the test?

Reply:
I looked at implementation of FileDownload function of FIle Api and there was passed true, so I assumed this is correct way. But I agree taht better to add third option which will handle other cases.